ui/src/app/workflows/components/workflow-filters/workflow-filters.tsx

```diff
@@ -6,7 +6,13 @@ import {TagsInput} from '../../../shared/components/tags-input/tags-input';
 
 require('./workflow-filters.scss');
 
+export enum FilterType {
+    WORKFLOW,
+    ARCHIVED_WORKFLOW
+}
+
 interface WorkflowFilterProps {
+    type: FilterType;
```

Comment:
this creates tight coupling between the components - I think you need to decouple them

Reply:
update.